### PR TITLE
add script to recycle statefulset

### PR DESCRIPTION
**Pull request by xnulinu**

### CODE REVIEW

File: scripts/README.md [hunk 1]
```diff
@@ -13,6 +13,17 @@ usage: ./pv-resize.sh -c <cluster-name> -t <cluster-type> -n <namespace> -s <siz
   -h | --help            : Usage command
 ```
 
+## Recycle Statefulset 
+
+```bash
+usage: ./sts-recreate.sh -c <cluster-name> -t <cluster-type> -n <namespace>
+
+  -c | --cluster-name    : name of the cluster to resize the PV
+  -t | --cluster-type    : confluent platform type, supported value: kafka ksqldb controlcenter zookeeper schemaregistry connect
```
Comment (rohit2b) [inline scripts/README.md:22]: "confluent platform type" -> "confluent platform component"

File: scripts/sts-recreate.sh [hunk 2]
```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+set -C -e -o pipefail
+##
+## Only run this script if you want to recreate statefulset: forbidden issue, revision problems, etc
```
Comment (rohit2b) [inline scripts/sts-recreate.sh:4]: Copy editing suggestions. Does this still hold technically true?
## Only run this script if you want to recreate the statefulset: to deal with forbidden issues, 
revision problems, etc
## This script deletes the statefuset with cascading false (orphaning the pods) and forces 
the operator to recreate the statefulset once all the pods are recycled.

**Review by rohit2b — CHANGES_REQUESTED** [review]

File: scripts/sts-recreate.sh (hunk 2 above)
Comment (xnulinu) [reply scripts/sts-recreate.sh:4]: happens sometime if we need to recycle statefulset

Comment (xnulinu) [reply scripts/sts-recreate.sh]: fixed

**Review by rohit2b — APPROVED** [review]
LGTM!

**Pull request merged**